packages/react/src/components/Coachmark/Coachmark.test.tsx

```diff
@@ -1,12 +1,120 @@
-import * as path from 'path';
+import * as React from 'react';
+import { safeCreate } from '@fluentui/test-utilities';
+
 import { Coachmark } from './Coachmark';
+import * as path from 'path';
 import { isConformant } from '../../common/isConformant';
+import { resetIds } from '@fluentui/utilities';
+
+const ReactDOM = require('react-dom');
 
 describe('Coachmark', () => {
+  beforeEach(() => {
+    resetIds();
+  });
+
+  afterAll(() => {
+    resetIds();
+  });
+
+  // Conformance Tests:
   isConformant({
     Component: Coachmark,
     displayName: 'Coachmark',
     componentPath: path.join(__dirname, 'Coachmark.ts'),
     disabledTests: ['component-handles-classname', 'component-has-root-ref', 'component-handles-ref'],
   });
+
+  // Render Tests:
+  it('renders Coachmark (correctly)', () => {
+    const createPortal = ReactDOM.createPortal;
+    ReactDOM.createPortal = jest.fn(element => element);
+
+    safeCreate(<Coachmark target="test" />, component => {
+      const tree = component!.toJSON();
+      expect(tree).toMatchSnapshot();
+      ReactDOM.createPortal = createPortal;
+    });
+  });
+
+  it('renders Coachmark (className)', () => {
```

Comment:
Probably rather than using snapshot tests for most of these, it would be better to use standard tests (maybe in addition to snapshot tests). Snapshots kind of provide both too much and not enough information--there's so much included that it's hard to tell at a glance if the specific thing you wanted to test is working correctly.